README.md

```diff
@@ -1,6 +1,31 @@
+<div align="center">
```

Comment:
When previewing this Markdown file, I don't see that this `<div>` is having any effect. If that's the case should we just remove all `<div>`?